src/box/lua/schema.lua

```diff
@@ -1858,7 +1858,7 @@ local port_c = ffi.cast('struct port_c *', port)
 
 -- Helper function to check space:method() usage
 local function check_space_arg(space, method)
-    if type(space) ~= 'table' or space.id == nil then
+    if type(space) ~= 'table' or (space.id == nil and space.name == nil) then
```

Comment:
Nit: Should be `space._id_or_name == nil`? Same comment to `check_index_arg`.

Reply:
Box spaces don't have this field.